.github/workflows/abtesting.yml

```diff
@@ -116,3 +116,23 @@ jobs:
       run: |
         scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb \
           FirebaseABTesting.podspec --platforms=${{ matrix.target }} ${{ matrix.flags }}
+
+  archive-cron:
+    # Don't run on private repo.
+    if: github.event_name == 'schedule' && github.repository != 'FirebasePrivate/firebase-ios-sdk'
+
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        target: [ios, tvos, macos]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Setup project and archive
+      run: scripts/test_archiving.sh FirebaseABTesting ${{ matrix.target }} ArchiveOutputs/${{ matrix.target }}.xcarchive
+    - uses: actions/upload-artifact@v2
```

Comment:
What's the reason for uploading?  How will it be used?

Reply:
Good question! It can be downloaded and used to sign locally to do a size measurement, although that may not be a good enough reason to upload. Happy to remove it, especially if it's eating into a quota I'm unaware of.

We could also potentially just use the `.ipa` inside of the archive to do a rough size guesstimate as well without signing, that could be done on GHA.

Reply:
I believe there is some cost, so my preference would be to remove the upload until we have a definite need.

I like the idea of getting the size from this. At the very least, just adding an `ls -l` to the script could be low hanging fruit - and then we could expand it over time to upload and track ...